packages/dd-trace/src/dogstatsd.js

```diff
@@ -11,6 +12,7 @@ class Client {
     options = options || {}
 
     this._host = options.host || 'localhost'
+    this.__family = isIP(this._host)
```

Comment:
Double underscore is reserved for language features (ignore the other occurrences in the tracer, that's on us 😅 ).

Reply:
Single underscore ok?